Fix:

conftest.c:135:14: error: implicit declaration of function 'getpagesize' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

https://bugs.gentoo.org/898816
https://git.savannah.gnu.org/cgit/autoconf.git/commit/?id=028526149ee804617a302ccef22cc6adbda681b0
https://git.savannah.gnu.org/cgit/autoconf.git/commit/?id=33c26d2700f927432c756ccf7a4fc89403d35b95
--- configure.orig	2023-07-25 03:52:34.000000000 -0500
+++ configure	2023-07-27 01:42:39.000000000 -0500
@@ -4881,11 +4881,14 @@
 char *malloc ();
 #endif
 
-/* This mess was copied from the GNU getpagesize.h.  */
-#ifndef HAVE_GETPAGESIZE
+#ifndef getpagesize
 # ifdef _SC_PAGESIZE
-#  define getpagesize() sysconf(_SC_PAGESIZE)
-# else /* no _SC_PAGESIZE */
+#  define getpagesize() sysconf (_SC_PAGESIZE)
+# elif defined _SC_PAGE_SIZE
+#  define getpagesize() sysconf (_SC_PAGE_SIZE)
+# elif HAVE_GETPAGESIZE
+int getpagesize ();
+# else
 #  ifdef HAVE_SYS_PARAM_H
 #   include <sys/param.h>
 #   ifdef EXEC_PAGESIZE
@@ -4909,16 +4912,15 @@
 #  else /* no HAVE_SYS_PARAM_H */
 #   define getpagesize() 8192	/* punt totally */
 #  endif /* no HAVE_SYS_PARAM_H */
-# endif /* no _SC_PAGESIZE */
-
-#endif /* no HAVE_GETPAGESIZE */
+# endif
+#endif
 
 int
-main ()
+main (void)
 {
   char *data, *data2, *data3;
   const char *cdata2;
-  int i, pagesize;
+  long i, pagesize;
   int fd, fd2;
 
   pagesize = getpagesize ();
